FWCore/Integration/test/testSwitchProducerConditionalTaskEDAlias_cfg.py

```diff
@@ -65,3 +69,5 @@ def __init__(self, **kargs):
 
 process.ct2 = cms.ConditionalTask(process.intProducerAlias, process.intProducerDep, process.intProducerDepAlias, process.ct)
 process.p2 = cms.Path(process.intProducerDepAliasDep, process.ct2)
+
+process.ep = cms.EndPath(process.out)
```

Comment:
Was calling process.out just forgotten in the original code?

Reply:
Yes (although it doesn't really do much, but I added it now because my earlier intention was to have it in this test).